puppet/modules/kubernetes/templates/kube-apiserver.service.erb

```diff
@@ -92,6 +92,15 @@ ExecStart=<%= scope['kubernetes::_dest_dir'] %>/apiserver \
 <%- if scope['kubernetes::_service_account_key_file'] -%>
   --service-account-key-file=<%= scope['kubernetes::_service_account_key_file'] %> \
   --service-account-lookup \
+<% if @post_1_12 -%>
+ "--service-account-signing-key-file=<%= scope['kubernetes::_service_account_key_file'] %>" \
+ "--service-account-issuer=<%= @service_account_issuer %>" \
+<% if !@post_1_13 -%>
+  --service-account-api-audiences=<%= @_service_account_api_audiences.join(',') %> \
+<% else -%>
+  --api-audiences=<%= @_service_account_api_audiences.join(',') %> \
```

Comment:
I think this would be better readable
```suggestion
<% if @post_1_13 -%>
  --api-audiences=<%= @_service_account_api_audiences.join(',') %> \
<% else -%>
  --service-account-api-audiences=<%= @_service_account_api_audiences.join(',') %> \
```

Reply:
This is obviously me